### PR TITLE
Add document for releasing the library

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: docs/Release.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Fork
+* For the repo at https://github.com/microsoft/vcpkg
+  - If you already have a fork, make sure it is updated to the master branch
+
+  ![image](https://user-images.githubusercontent.com/7574801/128568379-deb14ed7-1ba3-4cb7-83d2-faecf8300f21.png)
+
+* Clone your fork locally `git clone https://github.com/<fork>/vcpkg.git`
+
+## Modify vcpkg/ports/CONTROL
```
Comment (halter73) [inline docs/Release.md:16]:
```suggestion
## Modify vcpkg/ports/microsoft-signalr/CONTROL
```

**Review by halter73 — APPROVED** [review]
Want to have me try to do the next one based on this?

**Comment by BrennanConroy** [comment]
> Want to have me try to do the next one based on this?

I had the same thought 😃

File: docs/Release.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Release a new version of the library
+
+## Create the release
+* Get on the correct commit that we want to release (normally tip of main branch)
+* `git tag <tag-name>` (previous release format: v0.1.0-alpha3)
+* `git push origin <tag-name>`
+
+## Fork
+* For the repo at https://github.com/microsoft/vcpkg
```
Comment (BrennanConroy) [inline docs/Release.md:9]:
```suggestion
* Fork the repo at https://github.com/microsoft/vcpkg
```

**Pull request merged**